atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraExpiringKeyValueService.java

```diff
@@ -91,25 +102,18 @@ public void multiPut(Map<TableReference, ? extends Map<Cell, byte[]>> valuesByTa
 
 
             Iterable<List<Entry<Cell, byte[]>>> partitions = partitionByCountAndBytes(sortedMap.entrySet(),
-                    getMultiPutBatchCount(), getMultiPutBatchSizeBytes(), table, new Function<Entry<Cell, byte[]>, Long>(){
-
-                @Override
-                public Long apply(Entry<Cell, byte[]> entry) {
-                    long totalSize = 0;
-                    totalSize += entry.getValue().length;
-                    totalSize += Cells.getApproxSizeOfCell(entry.getKey());
-                    return totalSize;
-                }});
-
+                    getMultiPutBatchCount(), getMultiPutBatchSizeBytes(), table, entry -> {
+                        long totalSize = 0;
```

Comment:
Nit: Above, you replaced a similar code block with a one-liner.  Would it be cleaner to also do something like `entry -> entry.getValue().length + Cells.getApproximateSizeOfCell(entry.getKey())` here?